Sources/Kitura/RouterResponse.swift

```diff
@@ -238,9 +241,12 @@ public class RouterResponse {
     /// is the alphanumeric characters and `[]$._`).
     /// - Returns: this RouterResponse.
     public func send(jsonp: JSON, callbackParameter: String = "callback") throws -> RouterResponse {
+        if RouterResponse.sanitizeJSIdentifierRegex == nil {
```

Comment:
This is a bad way to initialize a Static field. It is NOT thread safe.

A better way to do this is is something similar to the way the static field oneTime in KituraNet.ClientRequest is initialized.